backend/src/v5/middleware/dataConverter/schemas/groups.js

```diff
@@ -143,6 +143,13 @@ const schema = (group, strict = false) => {
 };
 
 Group.validateSchema = (group) => schema(group, true).validate(group);
-Group.castSchema = (group) => schema(group).cast(group);
+Group.castSchema = (group) => {
+	const castedSchema = schema(group).cast(group);
```

Comment:
It doesn't make a lot of sense to anyone who's reading this to see that if `updatedAt` is not a number then it should be taking the `createdAt` value 😆 

It's probably better to just prefill the value before casting:

````js
const groupToCast = {...group};
groupToCast.updatedAt = groupToCast.updatedAt || groupToCast.createdAt;
return schema(group).cast(group);
````

Reply:
sure that makes sense :P